src/contracts/pods/EigenPodManager.sol

```diff
@@ -51,17 +60,36 @@ contract EigenPodManager is Initializable, OwnableUpgradeable, Pausable, IEigenP
 
     /// @notice Oracle contract that provides updates to the beacon chain's state
     IBeaconChainOracle public beaconChainOracle;
-    
+
     /// @notice Pod owner to deployed EigenPod address
     mapping(address => IEigenPod) public ownerToPod;
 
+    /// @notice Mapping: podOwner => cumulative number of queued withdrawals of beaconchainETH they have ever initiated. only increments (doesn't decrement)
+    mapping(address => uint256) public numWithdrawalsQueued;
+
```

Comment:
inserting this mapping here will mess up the storage layout and explode the system when we upgrade.
it needs to go after the existing storage

Reply:
yeah whoops, i thought I placed all the new storage correctly, guess I missed this one